src/views/Resource/RefineRegistry.vue

```diff
@@ -277,7 +277,10 @@ export default {
       return false;
     },
     resultsButtonActive() {
-      return this.getFairSharingRecords.length > 0;
+      if (this.getFairSharingRecords) {
```

Comment:
if (this.getFairSharingRecords && this.getFairSharingRecords.length) {
return true
}
return false